COLLABORATOR_GUIDE.md

```diff
@@ -618,7 +618,7 @@ It is important to mention that there are some rules on our Vercel Deployments s
 
 - Branches starting with `dependabot` (Dependabot Automated PRs) or `gh` (GitHub Merge Queues) are not deployed to Vercel.
 - Vercel Deployments are triggered for all other branches during `push` activities.
-- We have a custom install script that executes `npm ci --omit=dev` (the same way we do on our CI Pipelines)
+- We have a custom install script that executes `pnpm install --prod` (the same way we do on our CI Pipelines)
```

Comment:
```suggestion
- We have a custom install script that executes `pnpm install --prod --frozen-lockfile` (the same way we do on our CI Pipelines)
```
If my other suggestions are accepted, we probably want to include it here too for similar clarity reasons